src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,77 @@
+package core.basesyntax.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitTransaction {
+    public static final Map<String, Operation> operations = new HashMap<>();
```

Comment:
it's better to move this Map inside of enum and create a separate method that will be returning an operation by letter, in case letter isn't valid you should throw an exception[